tests/framework/integration/cluster.go

```diff
@@ -1693,5 +1693,20 @@ func (c *Cluster) MustNewMember(t testutil.TB, resp *clientv3.MemberAddResponse)
 	}
 	m.InitialPeerURLsMap[m.Name] = types.MustNewURLs(resp.Member.PeerURLs)
 	c.Members = append(c.Members, m)
+
+	// c.mustNewMember(t) creates a new member without using resp.Member.PeerURLs.
+	// We should override PeerURLs and PeerListeners set by mustNewMember.
+	urls := types.MustNewURLs(resp.Member.PeerURLs)
+	m.PeerURLs = urls
+	var listeners []net.Listener
+	for _, url := range urls {
+		l, err := transport.NewListener(url.Host, url.Scheme, nil)
```

Comment:
The last parameter always `nil`? What if it's HTTPS communication?

Reply:
Added `tlsinfo *transport.TLSInfo` param to `MustNewMember()` in case callers need it.